framework/JobHandler.py

```diff
@@ -349,6 +355,7 @@ def reAddJob(self, runner):
         self.__queue.append(runner)
       else:
         self.__clientQueue.append(runner)
+      runner.trackTime('queue')
```

Comment:
shouldn't we track the time only in case the profile flag is True?

Reply:
I can probably make that happen, yeah.